src/ripple/ripple.js

```diff
@@ -109,25 +110,52 @@
       }
       this.setRippleXY(x, y);
       this.setRippleStyles(true);
-      requestAnimationFrame(this.animFrameHandler.bind(this));
+
+      if (this.timeout) {
+        window.clearTimeout(this.timeout);
```

Comment:
Probably `clearTimeout` instead of `window.clearTimeout`, similarly to `requestAnimationFrame`.